llvm/test/Analysis/Lint/const-store.ll

```diff
@@ -0,0 +1,42 @@
+; RUN: not opt --mtriple=amdgcn --passes=lint --lint-abort-on-error %s -o /dev/null |& FileCheck %s
+; RUN: opt --mtriple=amdgcn --mcpu=gfx1030 --passes=lint %s -o /dev/null |& FileCheck %s --check-prefixes=CHECK,CHECK0
+; RUN: opt --mtriple=x86_64 --passes=lint --lint-abort-on-error %s -o /dev/null |& FileCheck %s --allow-empty --check-prefix=NOERR
```

Comment:
Should use -disable-output instead of -o /dev/null. Does |& work on all shells? Usually use 2>&1